nerfstudio/data/datamanagers/base_datamanager.py

```diff
@@ -457,19 +456,14 @@ def create_eval_dataset(self) -> TDataset:
             scale_factor=self.config.camera_res_scale_factor,
         )
 
-    def _get_pixel_sampler(self, dataset: TDataset, *args: Any, **kwargs: Any) -> PixelSampler:
+    def _get_pixel_sampler(self, dataset: TDataset, num_rays_per_batch: int) -> PixelSampler:
         """Infer pixel sampler to use."""
-        if self.config.patch_size > 1:
-            return PatchPixelSampler(*args, **kwargs, patch_size=self.config.patch_size)
```

Comment:
I think these two lines are very important and the current code will break this logic. Can you look into that @AdamRashid96?